meta-balena-common/recipes-core/balena-rollback/files/rollback-parse-bootloader

```diff
@@ -29,9 +29,9 @@ fi
 if [ -f "${RESIN_BOOT_MOUNTPOINT}/resinOS_uEnv.txt" ]; then
 	BOOTLOADER_FILE="${RESIN_BOOT_MOUNTPOINT}/resinOS_uEnv.txt"
 else
-	grub_cfg=$(find "${RESIN_BOOT_MOUNTPOINT}" -name grub.cfg)
-	if [ -f "${grub_cfg}" ]; then
-		BOOTLOADER_FILE="${grub_cfg}"
+	grubenv=$(find "${RESIN_BOOT_MOUNTPOINT}" -name grubenv)
```

Comment:
On long/medium term we want to unify bootloader configuration (uboot and grub) in such a way that they can both read the same data. When that is done, we will stop sed-ing grub_cfg. Let's handle that separately and leave the current setup for now.

Reply:
Dropped